src/Centreon/Domain/Engine/EngineConfiguration.php

```diff
@@ -93,7 +93,9 @@ public function getIllegalObjectNameCharacters(): ?string
      */
     public function setIllegalObjectNameCharacters(?string $illegalObjectNameCharacters): EngineConfiguration
     {
-        $this->illegalObjectNameCharacters = $illegalObjectNameCharacters;
+        // 👉️ Proposition temporaire de deprecation
```

Comment:
Write comments in English

Reply:
Yes, this is about to be discussed tomorrow morning because I have things to ask about it.
It won't be present in the code at the end.